format.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eo pipefail
+
+format() {
+    # TODO: Automatically add license to the top of each file.
+    PYTHON_FILES=()
+    while IFS=$'\n' read -r line; do PYTHON_FILES+=("$line"); done \
+    < <($1 -- '*.py')
+    for i in "${!PYTHON_FILES[@]}"; do
+        # If file does not exist move on (e.g. 'git mv')
+        if [ ! -f "${PYTHON_FILES[i]}" ]; then
+            continue
+        fi
+
+        echo "Formatting...${PYTHON_FILES[i]}"
+
+        # NOTE: yapf->autoflake->isort order needs to be consistent with check_format.sh
+        yapf -i "${PYTHON_FILES[i]}" && autoflake --in-place --remove-all-unused-imports --remove-unused-variables ${PYTHON_FILES[i]} --exclude=__init__.py
```

Comment:
I don't think we have yapf in the requirements.txt I got the error 

Formatting...generative_data_prep/__main__.py
./format.sh: line 18: yapf: command not found

I think this will also be an issue for our ./test.sh file - pydocstyle and stuff is not in our requirements.txt